doc/ref/grplib.xml

```diff
@@ -587,7 +587,7 @@ of i.m.f.&nbsp;subgroups  of <M>GL_4(&ZZ;)</M>   with representative subgroups
 \ldots,  G_6</M> of isomorphism types  <M>G_1 \cong W(F_4)</M>, <M>G_2 \cong D_{12}
 \wr C_2</M>, <M>G_3 \cong  G_4 \cong C_2 \times  S_5</M>, <M>G_5 \cong W(B_4)</M>, and
 <M>G_6  \cong (D_{12}  </M><C>Y</C><M>  D_{12})   \!:\! C_2</M>.  The    corresponding
-<M>&QQ;</M>-classes, <M>R_1, \ldots, R_6</M>, say, are pairwise different except that
+<M>&QQ;</M>-classes, call them <M>R_1, \ldots, R_6</M>, are pairwise different except that
```

Comment:
I don’t like the “call them” formulation here, to me it’s weirder than the “say” formulation.

My suggestions of something that sounds a bit better:

```suggestion
<M>&QQ;</M>-classes, which we denote <M>R_1, \ldots, R_6</M>, are pairwise different except that
```
```suggestion
<M>&QQ;</M>-classes, let us call them <M>R_1, \ldots, R_6</M>, are pairwise different except that
```
```suggestion
<M>&QQ;</M>-classes, which we call <M>R_1, \ldots, R_6</M>, are pairwise different except that
```
```suggestion
<M>&QQ;</M>-classes, <M>R_1, \ldots, R_6</M>, are pairwise different except that
```